Source/WebKit/Configurations/ModelService.xcconfig

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2019-2022 Apple Inc. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY APPLE INC. ``AS IS'' AND ANY
```

Comment:
This is not the correct license text, seeing that it lacks `AND ITS CONTRIBUTORS` (see https://webkit.org/licensing-webkit/).

Reply:
Good catch, fixed!